### PR TITLE
Detailed explanation of level

### PR DESCRIPTION
**Pull request by joinimran**
As the user suggested, I have applied detailed descriptions of levels. 

Problem: https://github.com/MicrosoftDocs/windows-itpro-docs/issues/4577

### CODE REVIEW

File: windows/security/threat-protection/windows-defender-antivirus/specify-cloud-protection-level-windows-defender-antivirus.md [hunk 1]
```diff
@@ -37,12 +37,9 @@ You can specify the level of cloud-protection offered by Windows Defender Antivi
 4. Select **Properties**, select **Settings: Configure**, and then select **Windows Defender Antivirus**.
 5. On the **File Blocking Level** switch, select one of the following:
 
-    1. **High** to provide a strong level of detection
-    2. **High +** to apply additional protection measures
-    3. **Zero tolerance** to block all unknown executables
-
-        > [!WARNING]
-        > While unlikely, setting this switch to **High** might cause some legitimate files to be detected. The **High +** setting might impact client performance. We recommend you set this to the default level (**Not configured**).
+    1. **High** Applies a strong level of detection.
```
Comment (illfated) [inline windows/security/threat-protection/windows-defender-antivirus/specify-cloud-protection-level-windows-defender-antivirus.md]: Add a colon to indicate that the following sentence is the description of the preceding keyword.
```suggestion
    1. **High**: Applies a strong level of detection.
```

File: windows/security/threat-protection/windows-defender-antivirus/specify-cloud-protection-level-windows-defender-antivirus.md [hunk 2]
```diff
@@ -37,12 +37,9 @@ You can specify the level of cloud-protection offered by Windows Defender Antivi
...
+    2. **High +** Uses the high level and applies additional protection measures(may impact client performance).
```
Comment (illfated) [inline windows/security/threat-protection/windows-defender-antivirus/specify-cloud-protection-level-windows-defender-antivirus.md]: Same as above + additional adjustments:
```suggestion
    2. **High +**: Uses the **High** level and applies additional protection measures (may impact client performance).
```

File: windows/security/threat-protection/windows-defender-antivirus/specify-cloud-protection-level-windows-defender-antivirus.md [hunk 3]
```diff
@@ -37,12 +37,9 @@ You can specify the level of cloud-protection offered by Windows Defender Antivi
...
+    3. **Zero tolerance** Block all unknown executables.
```
Comment (illfated) [inline windows/security/threat-protection/windows-defender-antivirus/specify-cloud-protection-level-windows-defender-antivirus.md]: (add colon)
```suggestion
    3. **Zero tolerance**: Block all unknown executables.
```

File: windows/security/threat-protection/windows-defender-antivirus/specify-cloud-protection-level-windows-defender-antivirus.md [hunk 4]
```diff
@@ -65,14 +62,13 @@ For more information about Intune device profiles, including how to create and c
 
 1.  Double-click the **Select cloud protection level** setting and set it to **Enabled**. Select the level of protection:  
     1.  Setting to **Default Windows Defender Antivirus blocking level** provides strong detection without increasing the risk of detecting legitimate files.
```
Comment (illfated) [inline windows/security/threat-protection/windows-defender-antivirus/specify-cloud-protection-level-windows-defender-antivirus.md]: Based on the preceding sentence, the following lines should be a list and does not need "Selecting" or "Setting":
```suggestion
     1.  **Default Windows Defender Antivirus blocking level** provides strong detection without increasing the risk of detecting legitimate files.  
```

File: windows/security/threat-protection/windows-defender-antivirus/specify-cloud-protection-level-windows-defender-antivirus.md [hunk 5]
```diff
@@ -65,14 +62,13 @@ For more information about Intune device profiles, including how to create and c
 
 1.  Double-click the **Select cloud protection level** setting and set it to **Enabled**. Select the level of protection:  
     1.  Setting to **Default Windows Defender Antivirus blocking level** provides strong detection without increasing the risk of detecting legitimate files.  
-    2.  Setting to **High blocking level** applies a strong level of detection. 
-    3. **High + blocking level** applies additional protection measures.
+    2.  Setting to **High blocking level** applies a strong level of detection while optimizing client performace (greater chances of false positives).
```
Comment (illfated) [inline windows/security/threat-protection/windows-defender-antivirus/specify-cloud-protection-level-windows-defender-antivirus.md]: (same as above)
```suggestion
    2.  **High blocking level** applies a strong level of detection while optimizing client performace (greater chances of false positives).
```

File: windows/security/threat-protection/windows-defender-antivirus/specify-cloud-protection-level-windows-defender-antivirus.md [hunk 6]
```diff
@@ -65,14 +62,13 @@ For more information about Intune device profiles, including how to create and c
...
+    3. **High + blocking level** applies additional protection measures(may impact client performance and increase rsik of false positives).
```
Comment (illfated) [inline windows/security/threat-protection/windows-defender-antivirus/specify-cloud-protection-level-windows-defender-antivirus.md]: (minor adjustments)
```suggestion
    3. **High + blocking level** applies additional protection measures (may impact client performance and increase risk of false positives).
```

File: windows/security/threat-protection/windows-defender-antivirus/specify-cloud-protection-level-windows-defender-antivirus.md [hunk 7]
```diff
@@ -65,14 +62,13 @@ For more information about Intune device profiles, including how to create and c
...
     4. **Zero tolerance blocking level** blocks all unknown executables.
 
         > [!WARNING]
-        > While unlikely, setting this switch to **High** might cause some legitimate files to be detected (although you will have the option to unblock or dispute that detection). The **High +** setting might impact client performance. We recommend you set this to the default level (**Not configured**).
-
-1. Click **OK**.
+        > While unlikely, setting this switch to **High** and **High +** might cause some legitimate files to be detected (although you will have the option to unblock or dispute that detection).
```
Comment (illfated) [inline windows/security/threat-protection/windows-defender-antivirus/specify-cloud-protection-level-windows-defender-antivirus.md]: (grammatical improvements)
```suggestion
        > While unlikely, setting this switch to **High** or **High +** may cause some legitimate files to be detected (although you will have the option to unblock or dispute that detection).
```

**Comment by jvsam** [comment]
Thanks @illfated. @joinimran, please review the suggested changes so that @nenonix or @JohanFreelancer9 can do the final copy review.

**Comment by joinimran** [comment]
Done, thanks @illfated for all your contribution.

File: windows/security/threat-protection/windows-defender-antivirus/specify-cloud-protection-level-windows-defender-antivirus.md [hunk 8]
```diff
@@ -37,12 +37,9 @@ You can specify the level of cloud-protection offered by Windows Defender Antivi
...
 
-    1. **High** to provide a strong level of detection
-    2. **High +** to apply additional protection measures
-    3. **Zero tolerance** to block all unknown executables
-
-        > [!WARNING]
-        > While unlikely, setting this switch to **High** might cause some legitimate files to be detected. The **High +** setting might impact client performance. We recommend you set this to the default level (**Not configured**).
+    1. **High**: Applies a strong level of detection.
+    2. **High +**: Uses the **High** level and applies additional protection measures (may impact client performance).
+    3. **Zero tolerance**: Block all unknown executables.
```
Comment (JohanFreelancer9) [inline windows/security/threat-protection/windows-defender-antivirus/specify-cloud-protection-level-windows-defender-antivirus.md]:
```suggestion
    3. **Zero tolerance**: Blocks all unknown executables.
```

File: windows/security/threat-protection/windows-defender-antivirus/specify-cloud-protection-level-windows-defender-antivirus.md [hunk 9]
```diff
@@ -64,15 +61,14 @@ For more information about Intune device profiles, including how to create and c
 5.  Expand the tree to **Windows components > Windows Defender Antivirus > MpEngine**.
 
 1.  Double-click the **Select cloud protection level** setting and set it to **Enabled**. Select the level of protection:  
-    1.  Setting to **Default Windows Defender Antivirus blocking level** provides strong detection without increasing the risk of detecting legitimate files.  
-    2.  Setting to **High blocking level** applies a strong level of detection. 
-    3. **High + blocking level** applies additional protection measures.
+     1.  **Default Windows Defender Antivirus blocking level** provides strong detection without increasing the risk of detecting legitimate files.
```
Comment (illfated) [inline windows/security/threat-protection/windows-defender-antivirus/specify-cloud-protection-level-windows-defender-antivirus.md]: Looks like I have added an extra space here in one of my suggestions. That was not intended.
```suggestion
    1.  **Default Windows Defender Antivirus blocking level** provides strong detection without increasing the risk of detecting legitimate files.  
```

**Comment by joinimran** [comment]
Done.

**Comment by jvsam** [comment]
Hello @Dansimp, this pull request has been copy edited and is ready for your final review and approval. The descriptions are now in sync with the information the user sees on the actual screens (Intune & Group Policy Management Editor). Please let us know if there are outstanding issues to be resolved prior to merge. Thank you.

**Comment by illfated** [comment]
Rich Text Diff view, for easier overview of the changes:
https://github.com/MicrosoftDocs/windows-itpro-docs/pull/5068/files?short_path=c20f56c#diff-c20f56c413c3550eccd8b260606a30dc

**Comment by jvsam** [comment]
Hi @Dansimp, just a follow-up on the status of this PR. Kindly review when you have time. Thank you.

**Comment by jvsam** [comment]
As always, thank you @Dansimp for merging this PR. Much appreciated!

**Pull request merged**